examples/geometry_optimization.jl

```diff
@@ -0,0 +1,84 @@
+# # Geometry optimization
+#
+# We use the DFTK and Optim packages in this example to compute numerically the bond length
```

Comment:
"compute numerically" sounds strange to a chemist, better: "to optimise the bond length" or "to find the minimal-energy bond length"